app/helpers/location_helper.rb

```diff
@@ -21,12 +21,12 @@ def self.adapt_location_iq_response(body)
                 end
 
     loc = {
-      name: body["display_name"],
+      name: normalize_name_aliases(body["display_name"]),
       geo_level: geo_level,
-      country_name: address[country_key] || "",
-      state_name: address[state_key] || "",
-      subdivision_name: address[subdivision_key] || "",
-      city_name: address[city_key] || "",
+      country_name: normalize_name_aliases(address[country_key] || ""),
+      state_name: normalize_name_aliases(address[state_key] || ""),
+      subdivision_name: normalize_name_aliases(address[subdivision_key] || ""),
+      city_name: normalize_name_aliases(address[city_key] || ""),
```

Comment:
did you consider running a migration to normalize the data that's already in prod?

Reply:
I've corrected the existing ones already but I do plan to do it again manually 👍